tests/deploy/values/ci-eks-config-2.yml

```diff
@@ -0,0 +1,23 @@
+elasticsearch:
+  volumeClaimTemplate:
+    storageClassName: gp2
+
+image:
+  repository: 704479110758.dkr.ecr.eu-west-2.amazonaws.com/cloudbeat-test
+  pullPolicy: Always
+
+cloudbeat:
+  deploy: false
+
+cloudbeatEks:
+  deploy: true
+
+standaloneAgent:
+  deploy: false
+
+testData:
+  marker: pre_merge
```

Comment:
I thought we aim for post_merge only for now?

Reply:
what about this comment?

Reply:
post_merge you mentioned is related to CI/CD process. Marker provided in values file is related to which collection of tests shall be executed. We need to rename markers in order to prevent confusion in the future.